javascript/react-js/state-and-props.md

```diff
@@ -232,6 +232,10 @@ For the state in functional components, it is a little bit different. Before the
 - How to handle state and props in a class component
 - Understanding setState()
 
+### Thinking in React
+
```

Comment:
I'm going to be honest, I feel like this part is sort of tacked on. This should instead be an assignment bullet point or an additional resource.

Reply:
added.